cli/server.go

```diff
@@ -207,15 +207,25 @@ func Server(dflags *codersdk.DeploymentFlags, newAPI func(context.Context, *code
 			)
 			defer closeTunnel()
 
-			// If the access URL is empty, we attempt to run a reverse-proxy tunnel
-			// to make the initial setup really simple.
+			// If the access URL is empty, we attempt to run a reverse-proxy
+			// tunnel to make the initial setup really simple.
 			if dflags.AccessURL.Value == "" {
 				cmd.Printf("Opening tunnel so workspaces can connect to your deployment. For production scenarios, specify an external access URL\n")
 				tunnel, tunnelErr, err = devtunnel.New(ctxTunnel, logger.Named("devtunnel"))
 				if err != nil {
 					return xerrors.Errorf("create tunnel: %w", err)
 				}
 				dflags.AccessURL.Value = tunnel.URL
+
+				if dflags.WildcardAccessURL.Value == "" {
+					u, err := parseURL(ctx, tunnel.URL)
+					if err != nil {
+						return xerrors.Errorf("parse tunnel url: %w", err)
+					}
+
+					// Suffixed wildcard access URL.
+					dflags.WildcardAccessURL.Value = fmt.Sprintf("*--%s", u.Hostname())
```

Comment:
Shouldn't this just be one hypen?

Reply:
acutaly looks like ti deosnt matter

Reply:
yeah it doesn't matter on the server side how many hyphens you use, but I used two here so it matches the two hyphens between each app component